.local/bin/workman

```diff
@@ -212,6 +231,29 @@ _find_assigned_for_task() {
     done
 }
 
+open() {
+    TASK="${1-}"
+    ASSIGNED="$(_find_assigned_for_task ${TASK})"
+    if [ ! -z $ASSIGNED ]; then
+        _open $TASK $ASSIGNED
+    else
+        echo "${RED}No task named ${TASK} found" >&2
+        echo "${YELLOW}Existing tasks:"
+        list
+        exit 1
+    fi
+}
+
+_open() {
```

Comment:
Could you add a short comment before the function describing what it does?

Reply:
This function will also need to re-source the `SESSION_REATTACH_COMMAND` option so it is defined when `TASK` is available.

Reply:
I have tested this locally, so it works, the command is available globally

Reply:
Huh, I would have thought that `SESSION_REATTACH_COMMAND` would have ended up with something like `rust/` because there was no `TASK` variable set when it was evaluated. If it works then fair enough.